loadtest/control/actions.go

```diff
@@ -726,6 +726,22 @@ func Reload(u user.User) UserActionResponse {
 		}
 	}
 
+	ver, err := u.Store().ServerVersion()
+	if err != nil {
+		return UserActionResponse{Err: NewUserError(err)}
+	}
```

Comment:
Given we can assume that version won't change at runtime I am thinking we could avoid some repetition by storing it in the controller struct when we read it the first time:

https://github.com/mattermost/mattermost-load-test-ng/blob/4c6f90ce00fbec2d54b800d2c51f13443d8878d0/loadtest/control/simulcontroller/controller.go#L77